api/candlepin-api-spec.yaml

```diff
@@ -4100,6 +4100,12 @@ paths:
         - Jobs
       operationId: scheduleJob
       security: []
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: object
```

Comment:
While implementing the spec tests for the new inactive consumer job, I ran into the following issue with OpenApi when using a new jobs client that extends the generated JobsApi class.
The issue is that the POST on the /jobs/schedule/{jobKey} endpoint does not have a request body defined and so an NPE occurs in the ApiClient.buildRequest method. Adding an object as the request body was the only way I could find to work around the issue.

**OpenAPI bug**
https://github.com/OpenAPITools/openapi-generator/issues/11833

```
InactiveConsumerCleanerJobTest > shouldDeleteInactiveConsumers() FAILED
    java.lang.NullPointerException: Parameter specified as non-null is null: method okhttp3.MediaType$Companion.parse, parameter $this$toMediaTypeOrNull
        at okhttp3.MediaType$Companion.parse(MediaType.kt)
        at okhttp3.MediaType.parse(MediaType.kt)
        at org.candlepin.ApiClient.buildRequest(ApiClient.java:1131)
        at org.candlepin.ApiClient.buildCall(ApiClient.java:1087)
        at org.candlepin.resource.JobsApi.scheduleJobCall(JobsApi.java:953)
        at org.candlepin.resource.JobsApi.scheduleJobValidateBeforeCall(JobsApi.java:965)
        at org.candlepin.resource.JobsApi.scheduleJobWithHttpInfo(JobsApi.java:1008)
        at org.candlepin.resource.JobsApi.scheduleJob(JobsApi.java:987)
        at org.candlepin.spec.bootstrap.client.JobsClient.scheduleJob(JobsClient.java:27)
        at org.candlepin.spec.InactiveConsumerCleanerJobTest.shouldDeleteInactiveConsumers(InactiveConsumerCleanerJobTest.java:93)
```

Reply:
Is adding the request body on the API spec the only way to work around this? It would be preferable if we can work around it on the client side somehow.

Reply:
Also, your branch needs rebasing since it has conflicts after https://github.com/candlepin/candlepin/pull/3324 was merged in master (and that one has a lot of changes on the java spec tests, so you'll have to adjust your spec test code)

Reply:
Awesome! I removed the request body and the endpoint works as expected with that version rolled back